charts/postgres-operator/values-crd.yaml

```diff
@@ -163,6 +163,8 @@ configLoadBalancer:
   enable_master_load_balancer: false
   # toggles service type load balancer pointing to the replica pod of the cluster
   enable_replica_load_balancer: false
+  # toggles service type load balancer pointing to the pooler pod of the cluster
+  enable_pooler_load_balancer: false
```

Comment:
We did not consider the master/replica in the pooler yet, but probably good idea to make this `enable_master_pooler_load_balancer`

Reply:
Wait, pooler works only with master nodes? Shouldn't it delegate write requests to master and read request to all nodes?

Reply:
This is not possible. This needs to be a conscious decision in your application. Supporting the pooler in front of the replicas will follow.

PGBouncer cannot decide this. Imagine a transaction spanning multiple commands and starting only with reads and ending with a write, where would you point it to.

Reply:
Oh, that makes sense. Renamed.

Reply:
pgpool do support routing read-only request to slave.